### PR TITLE
Fix mobile auto select

### PR DESCRIPTION
**Pull request by codengoo**
Mobile App: Channel Mgmt > UI issues of viewing user's profile
[#2231](https://github.com/nccasia/mezon-fe/issues/2231)

Mobile App: Channel Mgmt > The magement options under user's profile are still display after that user was kicked
[#2232](https://github.com/nccasia/mezon-fe/issues/2232)

### CODE REVIEW

File: apps/mobile/src/app/temp-ui/MezonButton2/styles.tsx [hunk 1]
```diff
@@ -29,6 +29,18 @@ export const style = (colors: Attributes) =>
...
+		},
+
+		containerMd: {
+			padding: Metrics.size.l,
+		},
+
+		containerLg: {
+			padding: Metrics.size.l,
+		},
+
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/temp-ui/MezonButton2/styles.tsx]: using size

File: apps/mobile/src/app/temp-ui/MezonButton2/styles.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ export const style = (colors: Attributes) =>
 			display: 'flex',
 			flexDirection: 'row',
 			justifyContent: 'center',
-			borderRadius: 5,
+			borderRadius: 10,
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/temp-ui/MezonButton2/styles.tsx]: using size

File: apps/mobile/src/app/temp-ui/MezonButton2/index.tsx [hunk 3]
```diff
@@ -8,23 +8,37 @@ interface IMezonButton {
...
 		if (type === 'success') return styles.containerSuccess;
 		if (type === 'warning') return styles.containerWarning;
 		if (type === 'danger') return styles.containerDanger;
+		if (type === 'theme') return styles.containerTheme;
+		return {};
+	}
+
+	function renderContainerSize() {
+		if (size === "md") return styles.containerMd;
+		if (size === "lg") return styles.containerLg;
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/temp-ui/MezonButton2/index.tsx]: wrapper to useCallback

File: apps/mobile/src/app/screens/messages/NewMessage/index.tsx [hunk 4]
```diff
@@ -139,6 +126,7 @@ export const NewMessageScreen = ({ navigation }: { navigation: any }) => {
 				friendList={filteredFriendList}
 				handleFriendAction={handleFriendAction}
 				showAction={false}
+				onScroll={() => { Keyboard.dismiss() }}
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/messages/NewMessage/index.tsx]: Don't spam functions onScroll

File: apps/mobile/src/app/screens/home/homedrawer/components/UserProfile/index.tsx [hunk 5]
```diff
@@ -86,7 +86,8 @@ const UserProfile = React.memo(({ userId, user, onClose, checkAnonymous, message
 					<MezonAvatar
 						width={80}
 						height={80}
-						avatarUrl={userById?.user?.avatar_url || user?.avatar_url}
+						// @ts-ignore
+						avatarUrl={userById?.user?.avatar_url || user?.avatar_url || user?.avatarSm}
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/home/homedrawer/components/UserProfile/index.tsx]: fix type. Don't using ts-ignore

File: apps/mobile/src/app/temp-ui/MezonButton2/index.tsx [hunk 6]
```diff
@@ -8,23 +8,37 @@ interface IMezonButton {
...
 	const { themeValue } = useTheme();
 	const styles = style(themeValue);
 
 	function renderContainerStyle() {
 		if (type === 'success') return styles.containerSuccess;
 		if (type === 'warning') return styles.containerWarning;
 		if (type === 'danger') return styles.containerDanger;
+		if (type === 'theme') return styles.containerTheme;
+		return {};
+	}
```
Comment (sontung-nccvn) [inline apps/mobile/src/app/temp-ui/MezonButton2/index.tsx]: - wrap this with useMemo
- create enum for type
- use switch case instead

**Review by phuocnguyenncc — APPROVED** [review]

**Pull request merged**